profile/README.md

```diff
@@ -1,79 +1,58 @@
 
-### [Taxonomy](https://github.com/instructlab/taxonomy)
+Take a look at "lab-enhanced" models on the [InstructLab Hugging Face](https://huggingface.co/instructlab) page.
 
-The InstructLab method is driven by taxonomies, which are largely created manually and with care. This repository contains a taxonomy tree that will allow you to create models tuned with your data (enhanced via synthetic data generation) using the InstructLab method.
+## Get Started with InstructLab
 
-* [Repo](https://github.com/instructlab/taxonomy)
-* [Contributing](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md)
+* Check out the [Community README](https://github.com/instructlab/community/blob/main/README.md) to get started with using the project. (Contribution guides are included in this document.)  
+* You may wish to read through the [project's FAQ](https://github.com/instructlab/community/blob/main/FAQ.md) to get more familiar with all aspects of InstructLab.
+* You can find all the ways to collaborate with project maintainers and your fellow users of InstructLab beyond GitHub by visiting our [project collaboration](https://github.com/instructlab/community/blob/main/Collaboration.md) page.
 
-### [Command-line Interface](https://github.com/instructlab/instructlab)
+## Code of Conduct
 
-This command-line interface for InstructLab will allow you to create models tuned with data you provide using the InstructLab method on your laptop or workstation.
+Participation in the InstructLab community is governed by our [Code of Conduct](https://github.com/instructlab/community/blob/main/CODE_OF_CONDUCT.md)
 
-* [Repo](https://github.com/instructlab/instructlab)
-* [Contributing](https://github.com/instructlab/instructlab/blob/main/CONTRIBUTING/CONTRIBUTING.md)
+## Quick Links
 
-> [!IMPORTANT]
-> It is important to understand that running InstructLab on a laptop will give you a low-fidelity approximation of both synthetic data generation (using the `ilab generate` command)
-> and model instruction tuning (using the `ilab train` command, which uses QLoRA.) The quality of the results you get using these tools on a laptop will not be as high-fidelity as they might be using
-> a larger teacher model and a different training method. We have optimized InstructLab to enable community members with modest hardware to be able to use the technique. If you have more sophisticated
-> hardware, you can configure InstructLab to use a larger teacher model [such as Mixtral](https://huggingface.co/docs/transformers/model_doc/mixtral) in order to achieve higher-fidelity results.
-  
-## Model Training Infrastructure
+### Governance
 
-The infrastructure used to regularly train the model based on new contributions from the community is GPU intensive and generously donated and maintained by IBM.
+Overview of how InstructLab project [governance](https://github.com/instructlab/community/blob/main/governance.md) operates.
```

Comment:
```suggestion
See [here](https://github.com/instructlab/community/blob/main/governance.md) for an overview of how InstructLab project is governed.
```

Reply:
I think s/here/the governance document/ or s/here/the community repository/ would be a little nicer -- something other than "here"